exercises/all-your-base/all_your_base_test.rb

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env ruby
 require 'minitest/autorun'
 require_relative 'all_your_base'
 
-# Common test data version: 1.0.0 39de23b
+# Common test data version: 1.1.0 c4d8d95
```

Comment:
These 5 need reverting. The updates to tests which are not just removing the shebang line should not be included in this PR.

exercises/all-your-base/all_your_base_test.rb
exercises/bowling/bowling_test.rb 
exercises/hamming/hamming_test.rb 
exercises/pangram/pangram_test.rb 
exercises/rna-transcription/rna_transcription_test.rb